doom-modeline-segments.el

```diff
@@ -1024,6 +1024,18 @@ mouse-1: List all problems%s"
            (doom-modeline-spc)))
       "")))
 
+;;
+;; Word Count
+;;
+
+(doom-modeline-def-segment word-count
+  "The buffer word count.
+Displayed when in a major mode in `doom-modeline-continuous-word-count-modes'.
+Respects `doom-modeline-enable-word-count'."
+  (when (and doom-modeline-enable-word-count
+             (doom-modeline--active)
```

Comment:
I think this can be removed since it should be displayed on both active and inactive windows.

Reply:
Good nudge thanks — I hid it in inactive windows to fix this UI bug:

<img width="557" alt="Screen Shot 2019-11-24 at 7 32 46 AM" src="https://user-images.githubusercontent.com/4433943/69494805-e5fd9400-0e8d-11ea-950a-4b6124e00ba1.png">


Fixed more granularly in d4b4ea4 : 

<img width="525" alt="Screen Shot 2019-11-24 at 7 38 53 AM" src="https://user-images.githubusercontent.com/4433943/69494804-e433d080-0e8d-11ea-9572-9e39e74ad7ae.png">

Reply:
This is more clear.
```elisp
(propertize word-count 'face (if (doom-modeline--active)
                                 'mode-line
                              'mode-line-inactive)
```

Reply:
Updated in c87555d